### DISCUSSION TITLE
Using popover and link tags together cannot avoid default redirection

### DISCUSSION THREAD

**Discussion by imyuanli** [shadcn-ui/ui#3146] [General]
When I nest a link tag outside of a popover, clicking on the popover's trigger will redirect to the page. I cannot use bubbles to block it, nor can I block default events
```typescript
<Link href='/test'>
<div>testtesttest</div>
<Popover>
    <PopoverTrigger>
        <Button>
            Click
        </Button>
    </PopoverTrigger>
    <PopoverContent>
        testtesttest
    </PopoverContent>
</Popover>
</div>
```

**Comment by imyuanli** [comment]
I used a rather clumsy method
```typescript
<div onClick={(e)=>{ e.preventDefault()}}>
  <Popover ...> 
</div>
```

**Reply by samuelcastro** [reply]
This fixed my issue however, the `PopoverContent` takes a while to open up for some reason.

**Comment by imyuanli** [comment]
这是自动回复的邮件。您好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
